controllers/MarketplaceController.php

```diff
@@ -33,14 +33,17 @@ class MarketplaceController extends ContentContainerController
 
     public function actionIndex()
     {
+        $userGuide = false;
+
         $marketplaces = Marketplace::find()
             ->where(['space_id' => $this->contentContainer->id])
             ->orderBy(['created_at' => SORT_DESC])
             ->all();
 
         return $this->render('index', [
             'marketplaces' => $marketplaces,
-            'user' => $this->contentContainer
+            'user' => $this->contentContainer,
+            'userGuide' => empty($marketplaces) ? true : false
```

Comment:
why did you add `? true: false` here and not for others ? I think `empty($markplaces)` is sufficient